var/spack/repos/builtin/packages/moab/package.py

```diff
@@ -85,6 +85,11 @@ class Moab(AutotoolsPackage):
 
     patch('tools-492.patch', when='@4.9.2')
 
+    @run_before('configure')
+    def filter_configure(self):
+        if self.spec.satisfies('%fj'):
+            filter_file('-march=native', '', 'configure', string=True)
```

Comment:
I think this is worth doing always, since `-march=native` might conflicts with Spack ability to fine tune for micro-architectures:
```suggestion
    @run_before('configure')
    def remove_march_native(self):
        filter_file('-march=native', '', 'configure', string=True)
```
What do you think @adamjstewart ?